<launch>
  <node pkg="mlcc" type="calib_single_camera" name="calib_single_camera"
    args="$(find mlcc)/config/camera.yaml
          $(find mlcc)/config/scene_config.yaml
          $(find mlcc)/result"
    output="screen"/>

    <!-- distance threshold lower bound: -->
    <param name="distance_threshold_lower_bound" type="int" value="20"/>

    <!-- if using adaptive voxel -->
    <param name="use_adaptive_voxel" type="bool" value="1"/>
    <param name="use_rough_calib" type="bool" value="0"/>
    <param name="only_calibrate_rotation" type="bool" value="1"/>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find mlcc)/rviz_cfg/rosbag.rviz"/>
</launch>